rector.php

```diff
@@ -46,6 +48,14 @@
         '*/Source*',
         '*/Expected/*',
 
+        // stmts aware type
+        RenameParamToMatchTypeRector::class => [
+            __DIR__ . '/rules/DeadCode/Rector/If_/RemoveUnusedNonEmptyArrayBeforeForeachRector.php',
+        ],
+        AddMethodCallBasedStrictParamTypeRector::class => [
+            __DIR__ . '/rules/DeadCode/Rector/If_/RemoveUnusedNonEmptyArrayBeforeForeachRector.php',
+        ],
```

Comment:
this is not needed, I tested latest main branch without this skip and it is working ok, I will create PR to remove it.

Reply:
I had some local glitches with PHPStan cache.
Patches get stuck sometimes:)

Thanks for restoring 👌

Reply:
You may need to force remove it

```
php -r 'shell_exec("rm -rf " . sys_get_temp_dir() . "/rector_cached_files");';
php -r 'shell_exec("rm -rf " . sys_get_temp_dir() . "/cache/PHPStan");';
php -r 'shell_exec("rm -rf " . sys_get_temp_dir() . "/cache/nette.configurator");';
```